src/lib/mixer_module/mixer_module.cpp

```diff
@@ -322,6 +322,22 @@ bool MixingOutput::update()
 		return false;
 	}
 
+	if (_vehicle_land_detected_sub.updated()) {
+		vehicle_land_detected_s vehicle_land_detected;
+
+		if (_vehicle_land_detected_sub.copy(&vehicle_land_detected)) {
+			// disable airmode when landed to settle on uneven ground
+			if (_param_mc_airmode.get() > 0) {
```

Comment:
This check just changes runtime in case airmode is enabled right?

Reply:
The idea was to skip it entirely for bogus values, but it might be easier/cleaner to simply handle all possible valid values (disabled, roll_pitch, roll_pitch_yaw).